src/index.html

```diff
@@ -1,15 +1,207 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0,
+      minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="GET"
+      onsubmit="onSubmit()"
+    >
+
+      <fieldset class="inputBlock">
+        <legend>Personal information</legend>
+        <label
+          class="form-field"
+          for="SName">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            id="SName"
+            name="Surname">
+        </label>
```

Comment:
```suggestion
            id="SName"
            name="Surname"
          >
        </label>
```
Closing bracket should be on the newline when you use multi-line attributes